# generated by patch-package 6.4.8 on 2023-03-01 18:10:22
#
# command:
#   npx patch-package reselect
#
# declared package:
#   reselect: 4.1.7
#
diff --git a/node_modules/reselect/es/types.d.ts b/node_modules/reselect/es/types.d.ts
index a399f25..b3a38d6 100644
--- a/node_modules/reselect/es/types.d.ts
+++ b/node_modules/reselect/es/types.d.ts
@@ -5,7 +5,7 @@ export type { MergeParameters } from './versionedTypes';
  * @param Result The final result returned by the selector
  * @param Params All additional arguments passed into the selector
  */
-export type Selector<State = any, Result = unknown, Params extends never | readonly any[] = any[]> = [Params] extends [never] ? (state: State) => Result : (state: State, ...params: Params) => Result;
+export type Selector<State = any, Result = unknown, Params extends never | readonly any[] = never> = [Params] extends [never] ? (state: State) => Result : (state: State, ...params: Params) => Result;
 /** Selectors generated by Reselect have several additional fields attached: */
 export interface OutputSelectorFields<Combiner extends UnknownFunction> {
     /** The final function passed to `createSelector` */
